src/components/HURUmap/IndicatorTitle/Share.js

```diff
@@ -74,9 +87,24 @@ function Share({ title, geoCode, indicatorId, view, isCompare, ...props }) {
     <Grid container className={classes.root}>
       {shareData.map((social) => (
         <Grid item xs={4} key={social.name}>
-          <ShareButton name={social.name} url={url} {...social.props} />
+          {social.name === "Copy" ? (
+            <div className={classes.shareButton}>
+              <CopyToClipboard text={url} onCopy={handleOnCopy}>
+                <CopyIcon className={classes.exampleIcon} />
```

Comment:
```suggestion
                <SvgIcon
                  component={CopyIcon}
                  viewBox="0 0 28 28"
                  className={classes.copyIcon}
                /
```
1.When the SVG icon you're trying to use doesn't have a `viewBox` of `"0 0 24 24"`, you have to use `@material-ui/core/SvgIcon` component that allows you to set `viewBox` to the actual value in the SVG file.
2. Use descriptive names. What is `exampleIcon`?